### ISSUE TITLE
mpremote lists VID and PID backwards?

### ISSUE THREAD

**Issue by lurch** [micropython/micropython#7481]
I just noticed that `mpremote connect list` seemed to be displaying the PID _before_ the VID, so I had a look at the source and saw that https://github.com/micropython/micropython/blob/master/tools/mpremote/mpremote/main.py#L169 says
```python
                        p.device, p.serial_number, p.pid, p.vid, p.manufacturer, p.product
```
which seems a bit odd, as in my experience the USB VID (vendor id) is **always** displayed before the PID (product id).
I won't submit a PR though, in case this was a deliberate decision rather than just a typo?

**Comment by dpgeorge** [comment]
I tried to make the output match `lsusb`, so this is indeed a typo :)

**Comment by dpgeorge** [comment]
Fixed by d1decdfa937368bc83adf6829eba6743d75136d0
